charts/coturn/templates/certificate.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.tls.enabled }}
+{{- if .Values.tls.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: coturn-auto-certificate
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  issuerRef:
+    name: {{ .Values.tls.certManager.issuer.name }}
+    kind: {{ .Values.tls.certManager.issuer.kind }}
```

Comment:
```suggestion
    name: {{ .Values.tls.certManager.issuerName }}
    kind: {{ .Values.tls.certManager.issuerKind }}
```

Reply:
Fixed in [3cae51b](https://github.com/wireapp/wire-server/pull/2620/commits/3cae51b9080c3c316e57fa93fe396418c348f4c9)